internal/pipeline/testdata/invalidconfig/configfilenotfound/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/set-namespace:unstable
+    configPath: configMap.yaml
```

Comment:
I am guessing `configMap.yaml` is missing. Can we add a comment here. Also pl. use lower case for filename.

Reply:
done